### PR TITLE
[Keyboard] New Keyboard - BDE

### PR DESCRIPTION
**Pull request by kylemccreery**
## Description

Initial PR for adding the Big Dill Extended by MechWild

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechwild/bde/righty/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no	    # Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no 	# If you have attached the optional LED strips, change RGBLIGHT_ENABLE to yes
+
+LAYOUTS = ortho_3x14
```
Comment (zvecr) [inline keyboards/mechwild/bde/righty/rules.mk:28]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/mechwild/bde/righty/config.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Making it so you need to hold the modifier and other key for the time together, helps not accidentally hit activate the hold functions of bottom row*/
+#define IGNORE_MOD_TAP_INTERRUPT
+
+/* Setting tap term, helps not accidentally hit activate the hold functions of bottom row*/
+#define TAPPING_TERM 250
```
Comment (zvecr) [inline keyboards/mechwild/bde/righty/config.h:43]: As this is personal preference, It should only exist at the keymap level

```suggestion
```

File: keyboards/mechwild/bde/righty/config.h [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6D77 // mw = "MechWild"
+#define PRODUCT_ID      0x1702
+#define DEVICE_VER      0x0203
+#define MANUFACTURER    MechWild
+#define PRODUCT         BDE Righty
+#define DESCRIPTION     41-key ortholinear 30% with numpad
```
Comment (zvecr) [inline keyboards/mechwild/bde/righty/config.h:11]:
```suggestion
```

File: keyboards/mechwild/bde/lefty/config.h [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Making it so you need to hold the modifier and other key for the time together, helps not accidentally hit activate the hold functions of bottom row*/
+#define IGNORE_MOD_TAP_INTERRUPT
+
+/* Setting tap term, helps not accidentally hit activate the hold functions of bottom row*/
+#define TAPPING_TERM 250
```
Comment (zvecr) [inline keyboards/mechwild/bde/lefty/config.h:43]:
```suggestion
```

File: keyboards/mechwild/bde/lefty/config.h [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6D77 // mw = "MechWild"
+#define PRODUCT_ID      0x1701
+#define DEVICE_VER      0x0203
+#define MANUFACTURER    MechWild
+#define PRODUCT         BDE Lefty
+#define DESCRIPTION     41-key ortholinear 30% with southpaw numpad
```
Comment (zvecr) [inline keyboards/mechwild/bde/lefty/config.h:11]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Also, all *.c and *.h files need a gpl compliant licence header

File: keyboards/mechwild/bde/righty/rules.mk (hunk 1 above)
Comment (kylemccreery) [reply keyboards/mechwild/bde/righty/rules.mk:28]: I saw that removing the other bootloaders was suggested in the documentation but also that it was not that way in other board files. Thank you for clarifying the intent.

**Comment by kylemccreery** [comment]
@zvecr It would appear that I accidentally removed your request for qmk/collaborators that you put in as a followup to the requested changes. I was attempting to mark the change request as resolved and requested a rereview without realizing I should have dismissed the change request comment. Using github is still sort of foreign to me, I apologize.

File: keyboards/mechwild/bde/righty/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# BIG DILL EXTENDED
+
+![BIG DILL EXTENDED](https://i0.wp.com/mechwild.com/wp-content/uploads/2020/10/IMG_4458_web.jpg)
+
+A 41 key keyboard.
+
+Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
+Hardware Supported: BDE v2.3 Righty  
+Hardware Availability: [BIG DILL EXTENDED on MechWild](https://mechwild.com/product/big-dill-extended-bde/)
```
Comment (fauxpark) [inline keyboards/mechwild/bde/righty/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
* Hardware Supported: BDE v2.3 Righty
* Hardware Availability: [BIG DILL EXTENDED on MechWild](https://mechwild.com/product/big-dill-extended-bde/)
```

File: keyboards/mechwild/bde/lefty/readme.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# BIG DILL EXTENDED
+
+![BIG DILL EXTENDED](https://i0.wp.com/mechwild.com/wp-content/uploads/2020/10/IMG_4458_web.jpg)
+===
+
+A 41 key keyboard.
+
+Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
+Hardware Supported: BDE v2.3 Lefty  
+Hardware Availability: [BIG DILL EXTENDED on MechWild](https://mechwild.com/product/big-dill-extended-bde/)
```
Comment (fauxpark) [inline keyboards/mechwild/bde/lefty/readme.md:10]:
```suggestion
* Keyboard Maintainer: [Kyle McCreery](https://github.com/kylemccreery)
* Hardware Supported: BDE v2.3 Lefty
* Hardware Availability: [BIG DILL EXTENDED on MechWild](https://mechwild.com/product/big-dill-extended-bde/)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mechwild/bde/lefty/keymaps/fancy/config.h [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+ * This program is distributed in the hope that it will be useful, 
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ * GNU General Public License for more details. 
+ * 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */
+
+
```
Comment (drashna) [inline keyboards/mechwild/bde/lefty/keymaps/fancy/config.h:17]:
```suggestion
#pragma once
```

File: keyboards/mechwild/bde/lefty/keymaps/via/config.h [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+ * This program is distributed in the hope that it will be useful, 
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ * GNU General Public License for more details. 
+ * 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */
+
+
```
Comment (drashna) [inline keyboards/mechwild/bde/lefty/keymaps/via/config.h:17]:
```suggestion
#pragma once
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mechwild/bde/lefty/lefty.h [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */
+
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_3x14( \
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/lefty.h:22]: This isn't a true ortho 3x14, so this should just be `LAYOUT`.

```suggestion
#define LAYOUT( \
```

File: keyboards/mechwild/bde/lefty/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+#define FN3_B       LT(3, KC_B)
+#define CTL_Z       CTL_T(KC_Z)
+#define ALT_X       ALT_T(KC_X)
+#define ALT_N       ALGR_T(KC_N)
+#define CTL_M       RCTL_T(KC_M)
+#define SFT_ENT     RSFT_T(KC_ENT)
+#define WIN_C		LGUI_T(KC_C)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/default/keymap.c:31]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+    KC_7,    KC_8,    KC_9,    KC_0,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+    KC_4,    KC_5,    KC_6,    SFT_ENT, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,     KC_H,    KC_J,    KC_K,    KC_L,    KC_ESC,
+    KC_1,    KC_2,    KC_3,     	    CTL_Z,   ALT_X,   WIN_C,   KC_V,    FN2_BSPC, FN1_SPC, FN3_B,   ALT_N,   CTL_M,   SFT_ENT
+  ),
+
+  [1] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/default/keymap.c:37]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+    KC_F7,   KC_F8,   KC_F9,   KC_F10,  _______, _______, _______, RESET,   _______,  KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    KC_F4,   KC_F5,   KC_F6,   KC_TAB,  _______, _______, _______, _______, _______,  KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT,
+    KC_F1,   KC_F2,   KC_F3,       	    _______, _______, KC_LGUI, _______, KC_DEL,	  _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+  ),
+	
+  [2] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/default/keymap.c:43]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO,
+    KC_F11,  KC_F12,  _______,          _______, _______, _______, _______, _______,  KC_PSCR, KC_HOME, KC_PGDN, KC_PGUP, KC_END
+  ),
+
+  [3] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/default/keymap.c:49]:
```suggestion
  [3] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+
+uint8_t cur_dance(qk_tap_dance_state_t *state);
+void left_enter_finished(qk_tap_dance_state_t *state, void *user_data);
+void left_enter_reset(qk_tap_dance_state_t *state, void *user_data);
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c:55]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+    KC_7,    KC_8,    KC_9,    KC_0,           KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+    KC_4,    KC_5,    KC_6,    TD(left_enter), KC_A,    KC_S,    KC_D,    KC_F,    KC_G,     KC_H,    KC_J,    KC_K,    KC_L,    KC_ESC,
+    KC_1,    KC_2,    KC_3,     	           CTL_Z,   ALT_X,   WIN_C,   KC_V,    FN2_BSPC, FN1_SPC, FN3_B,   ALT_N,   CTL_M,   SFT_ENT
+  ),
+
+  [1] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c:61]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+    KC_F7,   KC_F8,   KC_F9,   KC_F10,  _______, _______, _______, RESET,   _______,  KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    KC_F4,   KC_F5,   KC_F6,   KC_TAB,  _______, _______, _______, _______, _______,  KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT,
+    KC_F1,   KC_F2,   KC_F3,       	    _______, _______, KC_LGUI, _______, KC_DEL,	  _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+  ),
+	
+  [2] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c:67]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c [hunk 18]
```diff
@@ -0,0 +1,122 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO,
+    KC_F11,  KC_F12,  _______,          _______, _______, _______, _______, _______,  KC_PSCR, KC_HOME, KC_PGDN, KC_PGUP, KC_END
+  ),
+
+[3] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/fancy/keymap.c:73]:
```suggestion
[3] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/via/keymap.c [hunk 19]
```diff
@@ -0,0 +1,57 @@
...
+#define CTL_Z       CTL_T(KC_Z)
+#define ALT_X       ALT_T(KC_X)
+#define ALT_N       ALGR_T(KC_N)
+#define CTL_M       RCTL_T(KC_M)
+#define SFT_ENT     RSFT_T(KC_ENT)
+#define WIN_C		LGUI_T(KC_C)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/via/keymap.c:32]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/via/keymap.c [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+    KC_7,    KC_8,    KC_9,    KC_0,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+    KC_4,    KC_5,    KC_6,    SFT_ENT, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,     KC_H,    KC_J,    KC_K,    KC_L,    KC_ESC,
+    KC_1,    KC_2,    KC_3,     	    CTL_Z,   ALT_X,   WIN_C,   KC_V,    FN2_BSPC, FN1_SPC, FN3_B,   ALT_N,   CTL_M,   SFT_ENT
+  ),
+
+  [1] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/via/keymap.c:38]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/via/keymap.c [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+    KC_F7,   KC_F8,   KC_F9,   KC_F10,  _______, _______, _______, RESET,   _______,  KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    KC_F4,   KC_F5,   KC_F6,   KC_TAB,  _______, _______, _______, _______, _______,  KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT,
+    KC_F1,   KC_F2,   KC_F3,       	    _______, _______, KC_LGUI, _______, KC_DEL,	  _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
+  ),
+	
+  [2] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/via/keymap.c:44]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/keymaps/via/keymap.c [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO,
+    KC_F11,  KC_F12,  _______,          _______, _______, _______, _______, _______,  KC_PSCR, KC_HOME, KC_PGDN, KC_PGUP, KC_END
+  ),
+
+  [3] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/keymaps/via/keymap.c:50]:
```suggestion
  [3] = LAYOUT(
```

File: keyboards/mechwild/bde/lefty/readme.md [hunk 23]
```diff
@@ -0,0 +1,16 @@
+# BIG DILL EXTENDED
+
+![BIG DILL EXTENDED](https://i0.wp.com/mechwild.com/wp-content/uploads/2020/10/IMG_4458_web.jpg)
+===
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/lefty/readme.md:4]:
```suggestion
```

File: keyboards/mechwild/bde/righty/righty.h [hunk 24]
```diff
@@ -0,0 +1,30 @@
...
+ * You should have received a copy of the GNU General Public License 
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */
+
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_3x14( \
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/righty.h:22]:
```suggestion
#define LAYOUT( \
```

File: keyboards/mechwild/bde/righty/keymaps/default/keymap.c [hunk 25]
```diff
@@ -0,0 +1,56 @@
...
+#define FN3_B       LT(3, KC_B)
+#define CTL_Z       CTL_T(KC_Z)
+#define ALT_X       ALT_T(KC_X)
+#define ALT_N       ALGR_T(KC_N)
+#define CTL_M       RCTL_T(KC_M)
+#define WIN_C		LGUI_T(KC_C)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/default/keymap.c:31]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/default/keymap.c [hunk 26]
```diff
@@ -0,0 +1,56 @@
...
+    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_0,   KC_7,    KC_8,    KC_9,       
+    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,     KC_H,    KC_J,    KC_K,    KC_L,    KC_ESC,  KC_ENT, KC_4,    KC_5,    KC_6,  
+    CTL_Z,   ALT_X,   WIN_C,   KC_V,    FN2_BSPC, FN1_SPC, FN3_B,   ALT_N,   CTL_M,   KC_RSFT,         KC_1,    KC_2,    KC_3
+  ),
+
+  [1] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/default/keymap.c:37]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/default/keymap.c [hunk 27]
```diff
@@ -0,0 +1,56 @@
...
+    _______, _______, _______, RESET,   _______,  KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS, KC_F10, KC_F7,   KC_F8,   KC_F9,
+    _______, _______, _______, _______, _______,  KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT, KC_TAB, KC_F4,   KC_F5,   KC_F6,
+    _______, _______, KC_LGUI, _______, KC_DEL,	  _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT,         KC_F1,   KC_F2,   KC_F3
+  ),
+	
+  [2] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/default/keymap.c:43]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/default/keymap.c [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+    _______, _______, _______, _______, _______,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,  KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,  KC_PSCR, KC_HOME, KC_PGDN, KC_PGUP, KC_END,           KC_F11,  KC_F12,  _______
+  ),
+
+  [3] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/default/keymap.c:49]:
```suggestion
  [3] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/via/keymap.c [hunk 29]
```diff
@@ -0,0 +1,56 @@
...
+#define FN3_B       LT(3, KC_B)
+#define CTL_Z       CTL_T(KC_Z)
+#define ALT_X       ALT_T(KC_X)
+#define ALT_N       ALGR_T(KC_N)
+#define CTL_M       RCTL_T(KC_M)
+#define WIN_C		LGUI_T(KC_C)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/via/keymap.c:31]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/via/keymap.c [hunk 30]
```diff
@@ -0,0 +1,56 @@
...
+    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,     KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_0,   KC_7,    KC_8,    KC_9,       
+    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,     KC_H,    KC_J,    KC_K,    KC_L,    KC_ESC,  KC_ENT, KC_4,    KC_5,    KC_6,  
+    CTL_Z,   ALT_X,   WIN_C,   KC_V,    FN2_BSPC, FN1_SPC, FN3_B,   ALT_N,   CTL_M,   KC_RSFT,         KC_1,    KC_2,    KC_3
+  ),
+
+  [1] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/via/keymap.c:37]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/via/keymap.c [hunk 31]
```diff
@@ -0,0 +1,56 @@
...
+    _______, _______, _______, RESET,   _______,  KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS, KC_F10, KC_F7,   KC_F8,   KC_F9,
+    _______, _______, _______, _______, _______,  KC_COMM, KC_DOT,  KC_SLSH, KC_SCLN, KC_QUOT, KC_TAB, KC_F4,   KC_F5,   KC_F6,
+    _______, _______, KC_LGUI, _______, KC_DEL,	  _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT,         KC_F1,   KC_F2,   KC_F3
+  ),
+	
+  [2] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/via/keymap.c:43]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mechwild/bde/righty/keymaps/via/keymap.c [hunk 32]
```diff
@@ -0,0 +1,56 @@
...
+    _______, _______, _______, _______, _______,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,  KC_LABK, KC_RABK, KC_QUES, KC_COLN, KC_DQUO, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,  KC_PSCR, KC_HOME, KC_PGDN, KC_PGUP, KC_END,           KC_F11,  KC_F12,  _______
+  ),
+
+  [3] = LAYOUT_ortho_3x14(
```
Comment (noroadsleft) [inline keyboards/mechwild/bde/righty/keymaps/via/keymap.c:49]:
```suggestion
  [3] = LAYOUT(
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Don't be intimidated by the number of suggestions; the only substantial thing is renaming the layout macro of both boards to `LAYOUT`, which requires updating all the keymaps – one change becomes twelve.

A GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/11047/files).

**Comment by kylemccreery** [comment]
@noroadsleft Thank you for your description of why it should be changed. Helps keep it framed properly for my future PRs. Changes applied.

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**